apps/aecore/test/aec_mining_tests.erl

```diff
@@ -40,10 +40,10 @@ mine_block_test_() ->
                  %% let_it_crash = generate_valid_test_data(TopBlock, 100000000000000),
                  Nonce = case aec_hard_forks:protocol_effective_at_height(Height + 1) of
                              ?ROMA_PROTOCOL_VSN    -> 1157794539819639234;
-                             ?MINERVA_PROTOCOL_VSN -> 17834638850274440624;
```

Comment:
Why these changes?

Reply:
This test checks we can find a solution with a known-to-be-working nonce. Since we change the block header info, this changes the hash of the block candidate we base the puzzle on. Hence "find a solution" test shall be adapted accordingly.